content/docs/user-guide/components/reference/atom/postfx-gradient-weight-modifier.md

```diff
@@ -5,16 +5,37 @@ description: 'Open 3D Engine (O3DE) PostFX Gradient Weight Modifier component re
 toc: true
 ---
 
-The **PostFX Gradient Weight Modifier** component modifies post-processing effects' (PostFX) weight based on another entity's gradient signal.
+The **PostFX Gradient Weight Modifier** component modifies the weight of post-processing effects (PostFX) based on a gradient provided by another entity.
```

Comment:
The **PostFX Gradient Weight Modifier** component modifies the blending weight of post-processing effects (PostFX) based on a gradient signal provided by another entity. (The gradient signal acts as a mask.)